### PR TITLE
[Access control] New operator "select query results not contain"

### PR DESCRIPTION
**Pull request by litvinovg**
# What does this pull request do?

Created operator SPARQL_SELECT_QUERY_RESULTS_NOT_CONTAIN to check if query results do not contain a value.
Renamed operator SPARQL_SELECT_QUERY_CONTAINS to SPARQL_SELECT_QUERY_RESULTS_CONTAIN.
Removed hardcoded variable name "resourceUri", instead add values from all variables returned by query.

# Interested parties
@chenejac @vivo-project/vivo-committers

### CODE REVIEW

File: home/src/main/resources/rdf/accessControl/firsttime/operators.n3 [hunk 1]
```diff
@@ -19,4 +19,7 @@ access-individual:StartsWith a access:Operator;
     access:id "STARTS_WITH" .
 
 access-individual:SparqlSelectQueryContains a access:Operator;
-    access:id "SPARQL_SELECT_QUERY_CONTAINS" .
+    access:id "SPARQL_SELECT_QUERY_RESULTS_CONTAIN" .
+
+access-individual:SparqlSelectQueryResultsNotContain a access:Operator;
```
Comment (chenejac) [inline home/src/main/resources/rdf/accessControl/firsttime/operators.n3:24]: Please align object names
access-individual:SparqlSelectQueryResultContain

**Review by chenejac — CHANGES_REQUESTED** [review]
@litvinovg well done. I have only one tiny comment. Also, I suggest not to add Not start with operation now, we can add that in the separate PR once when it is needed.

File: home/src/main/resources/rdf/accessControl/firsttime/operators.n3 (hunk 1 above)
Comment (litvinovg) [reply home/src/main/resources/rdf/accessControl/firsttime/operators.n3]: Done

Comment (litvinovg) [reply home/src/main/resources/rdf/accessControl/firsttime/operators.n3]: Fixed

**Review by chenejac — APPROVED** [review]
@litvinovg well done.

**Pull request merged**